modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/wal/reader/StandaloneGridKernalContext.java

```diff
@@ -677,7 +677,7 @@ private void setField(IgniteEx kernal, String name, Object val) throws NoSuchFie
 
     /** {@inheritDoc} */
     @Override public GridInternalSubscriptionProcessor internalSubscriptionProcessor() {
-        return null;
+        return new GridInternalSubscriptionProcessor(this);
```

Comment:
Does standalone instance support any connections? I don't see neither ClientProcessor, nor ability to work with other nodes.

Reply:
It's "for test only" change. I will move it to the tests.